test/resources/configuration_policy/role-policy-e2e-less.yaml

```diff
@@ -3,6 +3,6 @@ kind: Role
 metadata:
   name: role-policy-e2e
 rules:
-  - apiGroups: ["extensions", "apps"]
+  - apiGroups: ["apps", "extensions"]
     resources: ["deployments"]
     verbs: ["get", "list", "watch", "create", "delete","patch"]
```

Comment:
The verbs are getting re-sorted, too

Reply:
good catch, thanks

Reply:
```
"create", "delete", "get", "list", "patch", "watch"
```
It's weird the other set of three verbs isn't getting sorted, but maybe that's because it's not part of the policy?

Reply:
Oh, wait. Sometimes they are:
https://travis-ci.com/github/open-cluster-management/governance-policy-framework/jobs/526525182#L1438
And sometimes they aren't:
https://travis-ci.com/github/open-cluster-management/governance-policy-framework/jobs/526525182#L1117
I'm not familiar enough with the code to know whether these scenarios make sense or not.

Reply:
Oh, yeah--it's all about what's included in the policy. 🙂